doc/Options/zCOBOL_options.md

```diff
@@ -0,0 +1,13 @@
+# zCOBOL options
+
+Option          | Default | Description
+----------------|---------|------------
+@file           | NO      | Retrieve additional options from free form text file with default suffix OPT.  Options can be specified delimited by spaces on as many lines as requires.  All characters on a line following * are ignored as comments. The @file option can be nested.  The default path is the program path.
+CICS            | NO      | Parse COBOL EXEC CICS commands into z390 EXEC CICS compatible macro calls and also rename working storage to DFHEISTG.
+COMMENT         | YES     | Generate MLC comments showing original COBOL statement preceding each macro call statement.
+EXTEND          | YES     | Support up to 31 digits for DISPLAY (Z) and COMP-3 (P) type data items rather than limiting precision to ANSI 1985 standard of 18.
+FLOAT(DECIMAL)  | YES     | Set type of floating point for usage FLOAT-SHORT, FLOAT-LONG, and FLOAT-EXTENDED.  The choices are FLOAT(HEX) for Hexidecimal Floating Point (HFP) like COMP-1 and COMP-2, FLOAT(BINARY) for Binary Floating Point (BFP), or the default FLOAT(DECIMAL) for Decimal Floating Point (DFP).
```

Comment:
Line 9. "... for Hexidecimal ..." -> "... for Hexadecimal ...".

Reply:
Fixed aff28bd